bedrock/mozorg/templates/mozorg/what-is-a-browser.html

```diff
@@ -27,13 +27,13 @@ <h1 class="mzp-c-article-title">{{ self.page_title() }}</h1>
 
     <p>
     {% trans %}
-      The web is a vast and powerful tool. Over the course of a few decades the internet has changed the way we work, the way we play and the way we interact with one another. Depending on how it’s used it bridges nations, drives commerce, nurtures relationships, drives the innovation engine of the future and is responsible for more memes than we know what to do with.
+      The web is a vast and powerful tool. Over the course of a few decades the internet has changed the way we work, the way we play and the way we interact with one another. Depending on how it’s used, it bridges nations, drives commerce, nurtures relationships, drives the innovation engine of the future and is responsible for more memes than we know what to do with.
```

Comment:
> drives the innovation engine of the future and is responsible for more memes than we know what to do with.

Hmmm, to oxford comma or not to oxford comma, that is the question. 😆 (Not a blocker.)

Reply:
The Mozilla style guide says no Oxford comma. The views of the company are not my own :)